drf_hal_json/serializers.py

```diff
@@ -1,26 +1,79 @@
 from collections import defaultdict
 
+from rest_framework.reverse import reverse
+from rest_framework.utils.serializer_helpers import ReturnDict
+
 from drf_hal_json import EMBEDDED_FIELD_NAME, LINKS_FIELD_NAME, URL_FIELD_NAME
 from drf_hal_json.fields import HalHyperlinkedPropertyField, HalContributeToLinkField, \
     HalHyperlinkedSerializerMethodField
 from rest_framework.fields import empty, FileField, ImageField
 from rest_framework.relations import HyperlinkedIdentityField, HyperlinkedRelatedField, ManyRelatedField, RelatedField
-from rest_framework.serializers import BaseSerializer, HyperlinkedModelSerializer
+from rest_framework.serializers import BaseSerializer, HyperlinkedModelSerializer, ListSerializer
 from rest_framework.utils.field_mapping import get_nested_relation_kwargs
 
 
+class HalListSerializer(ListSerializer):
+
+    @property
+    def data(self):
+        # The parent class returns ReturnList
+        return ReturnDict(
+            {
+                LINKS_FIELD_NAME: {
+                    URL_FIELD_NAME: {
+                        'href': self.build_view_url()
```

Comment:
Or should I be using (e.g. [here](https://github.com/Artory/drf-hal-json/blob/master/drf_hal_json/pagination.py#L14)):

    {'href': self.request.build_absolute_uri()}

Reply:
I think we need to build URI:s the same way everywhere, so I would go with `self.request.build_absolute_uri()`, if that is enough for your use case.

Reply:
It certainly looks like it should work (and eliminates the need for some extra code).  At some point, we're going to need to update that code to address #27 but it's not urgent/blocker.  I'll push the change.